LOTRNames/assets/data/generation/human.names.json

```diff
@@ -15,13 +15,14 @@
+"Námo","Irmo"],
```

Comment:
Looks like this guy is on his own line, but he should be part of the line above.